lib/components/Textarea/Textarea.migration.md

```diff
@@ -0,0 +1,36 @@
+# Textarea Migration Guide
+
+## API Changes
+
+- `message={false}` is now `reserveMessageSpace={false}`.
```

Comment:
Lets capture the other renames here too:
`initialRows` is now `lines`
`maxRows` is now `lineLimit`
`countFeedback` needs to be refactored to `characterLimit`, previously took a function that returned the count and when to show it, now simply takes the limit